performance-evaluation/google_drive.py

```diff
@@ -0,0 +1,53 @@
+from constants import GDRIVE_FILE_NAME
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+from percol import Percol
+import readchar
+
+
+def select_from_list(folders):
+    si, so, se = "", "", ""
+    candd = [p[0] for p in folders]
+    with Percol(
+            actions=[],
+            descriptors={'stdin': si, 'stdout': so, 'stderr': se},
+            candidates=iter(candd)) as p:
+        p.loop()
+    results = p.model_candidate.get_selected_result()
+    ret = [pp for pp in folders if pp[0] == results]
+    return ret
+
+
+def google_authentication():
+    gauth = GoogleAuth()
+    if gauth.credentials is None:
+        gauth.LocalWebserverAuth()
+    elif gauth.access_token_expired:
+        gauth.Refresh()
+    else:
+        gauth.LoadCredentialsFile(GDRIVE_FILE_NAME)
+        gauth.Authorize()
+        gauth.SaveCredentialsFile(GDRIVE_FILE_NAME)
+    drive = drive_authentication(gauth)
+    return drive
+
+
+def drive_authentication(gauth):
+    drive = GoogleDrive(gauth)
```

Comment:
`return GoogleDrive(gauth)` is exactly the same. Not too much purpose to this functions since it's just an alias for `GoogleDrive` constructor but whatever

Reply:
Done